src/main/webapp/creatives/maps/simid_map.css

```diff
@@ -1,6 +1,26 @@
 /* Styling for maps creative */ 
 
 /*Styling for initial state */
+button {
```

Comment:
Could you include a screenshot of what the button looks like after these changes?

Reply:
yes, I will add a picture